server/app.ts

```diff
@@ -85,6 +83,15 @@ const getUserProgressData = async (req: Request) => {
 }
 
 const getAccountData = async (req: Request, res: Response) => {
+  // @ts-ignore
```

Comment:
I created the PR #966 to try to remove these `ts-ignore's`. Let me know that you think.

Reply:
Awesome, thank you... I'll have a look 👏